data/states/AK.yml

```diff
@@ -0,0 +1,136 @@
+code: AK
```

Comment:
@sarahgreen This is an example of how to structure the state files based on [FreeFrom's Alaska content](https://docs.google.com/document/d/1xJ760ylTB6RESVIz73GPwDUwHv4bIEizV6RCV1GlwKA/edit).